OnlinerTask/OnlinerTask.Data/Notifications/Technologies/Implementations/NetMqNotificator.cs

```diff
@@ -25,15 +25,21 @@ private static void SendTcpMessage(string text, string tcpString, string type, s
             using (var client = NetMQContext.Create().CreateRequestSocket())
             {
                 client.Connect(tcpString);
```

Comment:
Client doesn`t play any role in creating message, that`s why firstly we can create message and than configure client to send